arch/X86/X86MappingInsnOp_reduce.inc

```diff
@@ -5267,11 +5267,11 @@
 },
 {	/* X86_SHRD16mrCL, X86_INS_SHRD: shrd{w}	$dst, $src2, cl */
 	X86_EFLAGS_UNDEFINED_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_READ, CS_AC_READ, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, CS_AC_READ, 0 }
 },
 {	/* X86_SHRD16mri8, X86_INS_SHRD: shrd{w}	$dst, $src2, $src3 */
 	X86_EFLAGS_UNDEFINED_OF | X86_EFLAGS_MODIFY_SF | X86_EFLAGS_MODIFY_ZF | X86_EFLAGS_UNDEFINED_AF | X86_EFLAGS_MODIFY_PF | X86_EFLAGS_MODIFY_CF,
-	{ CS_AC_READ, CS_AC_READ, CS_AC_IGNORE, 0 }
+	{ CS_AC_READ | CS_AC_WRITE, CS_AC_READ, CS_AC_READ, 0 }
```

Comment:
Ok, I'm doing something wrong. When you write: "mri8" what does it means?
I interpreted it as:
- $dst is "m = memory"
- $src2 is "r = register"
- $src3 is "i8 = immediate 8 bit"